lib/src/filesaver.directive.ts

```diff
@@ -20,56 +20,89 @@ export class FileSaverDirective {
   @Output() readonly success = new EventEmitter<HttpResponse<Blob>>();
   @Output() readonly error = new EventEmitter<any>();
 
-  constructor(private el: ElementRef<HTMLButtonElement>, private fss: FileSaverService, private httpClient: HttpClient) {
+  private readonly destroy$ = new Subject<void>();
+
+  constructor(
+    private ngZone: NgZone,
```

Comment:
Hi @cipchk.
First, thanks for your work.
Second, I just wanted to bring awareness to this particular change that released with the 15.0.1 patch. Since this changes the constructor signature, this is a breaking change for applications that extend the `FileSaverDirective`. Not a big deal, but it would have been nice to see this documented somewhere, and possibly release this as a minor instead of a patch.
Best Regards ❤️

Reply:
Sorry, I'll attention to release version rules next time.